verification/curator-service/ui/src/components/Users.test.tsx

```diff
@@ -37,7 +37,14 @@ test('lists users', async () => {
         config: {},
         headers: {},
     };
-    mockedAxios.get.mockResolvedValueOnce(axiosResponse);
+    mockedAxios.get.mockImplementation((url) => {
+        if (url === '/api/users/roles') {
```

Comment:
can you use a switch on the url please and default to reject the promise? this way we're sure we're mocking the right calls.
switch (url) {
  case roles: resolve roles
  case users: resolve users
  default: reject promise
}

Reply:
Done